cmd/manager/main.go

```diff
@@ -25,20 +26,29 @@ import (
 	sourceCommit "github.com/operator-framework/operator-marketplace/pkg/version"
 
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
-	"github.com/operator-framework/operator-sdk/pkg/leader"
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
 
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kruntime "k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
 const (
-	leaderElectionConfigMapName = "marketplace-operator-lock"
+	// TODO(tflannag): Should this be configurable?
+	defaultLeaderElectionConfigMapName = "marketplace-operator-lock"
+	defaultRetryPeriod                 = 30 * time.Second
+	defaultRenewDeadline               = 60 * time.Second
+	defaultLeaseDuration               = 90 * time.Second
```

Comment:
These values mirror the ones that were recently updated in the downstream package-server-manager in order to meet the SNO GA requirements regarding API server downtime: https://github.com/openshift/operator-framework-olm/blob/master/cmd/package-server-manager/util.go#L14-L22.